build.sbt

```diff
@@ -155,15 +174,13 @@ lazy val cliBackup = project
   .in(file("cli-backup"))
   .settings(
     cliSettings,
-    name := s"$baseName-cli-backup",
-    libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-actor"  % akkaVersion,
-      "com.typesafe.akka" %% "akka-stream" % akkaVersion,
-      "com.monovore"      %% "decline"     % declineVersion
-    )
+    name := s"$baseName-cli-backup"
+  )
+  .dependsOn(coreCli   % "compile->compile;test->test",
```

Comment:
I am wondering if you need `test->test` here, do you reuse test classes in there?

Reply:
I don't use test-classes in coreCli but I would imagine at some point in the future we would have tests in there for the decline options/arguments (i.e. just the command line parsing part) and there is likely to be some common test functions there.

I will leave it there for now, we can always remove it later if we don't end up doing this.